### PR TITLE
Document examples for datetime functions

### PR DESCRIPTION
**Pull request by rigogsilva**
## Description

This PR adds date_trunc, extract, date_parse, and date_format functions example so when someone looks at the docs, they know exactly how to use it. This also improves docs when using trinols (language server) in VsCode.

## Screen Shots

Screenshot 2022-12-12 at 10 13 49 AM
Screenshot 2022-12-12 at 10 13 29 AM

### CODE REVIEW

**Comment by rigogsilva** [comment]
I did submit the CLA not to long ago. The bot probably doesn't have it yet.

File: docs/src/main/sphinx/functions/datetime.rst [hunk 1]
```diff
@@ -228,7 +228,16 @@ The above examples use the timestamp ``2001-08-22 03:04:05.321`` as the input.
 
 .. function:: date_trunc(unit, x) -> [same as input]
 
-    Returns ``x`` truncated to ``unit``.
+    Returns ``x`` truncated to ``unit``::
+
+        SELECT date_trunc('day' , TIMESTAMP '2022-10-20 05:10:00')
+        -- 2022-10-20 01:00:00.000
```
Comment (nineinchnick) [inline docs/src/main/sphinx/functions/datetime.rst:234]: Shouldn't the time be `00:00:00.000`, here and in the example for `month`?

Comment (rigogsilva) [reply docs/src/main/sphinx/functions/datetime.rst]: Yes. Updated it.

**Review by ebyhr — APPROVED** [review]

**Comment by ebyhr** [comment]
Merged, thanks!

**Pull request merged**